plugins/epos/config.ini

```diff
@@ -138,3 +138,14 @@ data_access_manual = ['https://epos-eu.github.io/EPOS-DCAT-AP/v3/#properties-for
 
 # Data model information
 terms_data_model = []
+#metadata standard
+metadata_standard = ['DCAT']
+
+
+[fairsharing]
+# username and password
+fairsharing_username = ['']
+
+fairsharing_password = ['']
+
+fairsharing_path = ['static/fairsharing_metadata_standards140224.json']
```

Comment:
no need of prefixing the properties with `farisharing_`, they are already under `[fairsharing]` category

Reply:
It is that way to make  the variable name is consistent between the two files. In the case that there is a need for other passwords I would prefer this name

Reply:
Updated in: [Changed names to reduce redundance in config.ini](https://github.com/EOSC-synergy/FAIR_eva/pull/142/commits/6789df37a26ce0a0cc2ada25929a82bbaa6c07e9)